### PR TITLE
add English translation about "How to configure global rate limiting"

### PR DESCRIPTION
**Pull request by wuseyu**
add English translation about "How to configure global rate limiting"

### CODE REVIEW

File: content/en/docs/v1.0/tutorials/global-rate-limit.md [hunk 1]
```diff
@@ -4,37 +4,37 @@ description:
 weight: 30
 ---
 
-## 安装示例程序
+## Install the sample program
```
Comment (zhaohuabing) [inline content/en/docs/v1.0/tutorials/global-rate-limit.md:7]: Sample -> demo 
一下同

File: content/en/docs/v1.0/tutorials/global-rate-limit.md [hunk 2]
```diff
@@ -4,37 +4,37 @@ description:
...
+You can test whichever program you want to choose.
 
 ```bash
 ➜  ~ kubectl get ns|grep meta
 meta-dubbo        Active   16m
 meta-thrift       Active   16m
 ```
 
-## 什么是全局限流
+## What is global_rate_limit
```
Comment (zhaohuabing) [inline content/en/docs/v1.0/tutorials/global-rate-limit.md:20]: What is global rate limiting

Comment (wuseyu) [reply content/en/docs/v1.0/tutorials/global-rate-limit.md:20]: OK，I will correct all my mistakes.

Comment (zhaohuabing) [reply content/en/docs/v1.0/tutorials/global-rate-limit.md:20]: @wuseyu 这个 PR 和 https://github.com/aeraki-mesh/website/pull/12 重复了。你能重新选一个 issuue 来翻译吗？ 谢谢

Comment (wuseyu) [reply content/en/docs/v1.0/tutorials/global-rate-limit.md:20]: OK,I wiill choose another one.

Comment (zhaohuabing) [reply content/en/docs/v1.0/tutorials/global-rate-limit.md:20]: 选择后可以在issue里comment说一下，我 assign 给你，这样就不会冲突了

**Pull request closed without merging**